cosmovisor/cmd/cosmovisor/cmd/version.go

```diff
@@ -16,6 +21,16 @@ func IsVersionCommand(arg string) bool {
 }
 
 // PrintVersion prints the cosmovisor version.
-func PrintVersion() {
+func PrintVersion() error {
 	fmt.Println("Cosmovisor Version: ", Version)
+
+	if err := Run([]string{"version"}); err != nil {
+		// Check the config and output details or any errors.
+		// Not using the cosmovisor.Logger in order to ignore any level it might have set,
+		// and also to not have any of the extra parameters in the output.
+		output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: time.Kitchen}
+		logger := zerolog.New(output).With().Timestamp().Logger()
+		cverrors.LogErrors(logger, "Can't run APP version", err)
```

Comment:
The reason this was done with the help text is that, a) it might be using an info level log line, and b) having the "module = cosmovisor" part looked confusing to me as part of the help text.

Here, though, it'd be appropriate to use the normal logger:
```
cverrors.LogErrors(cosmovisor.Logger, "Can't run APP version", err)
```